docs/www/data-transform.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Data transformation
+order: 0
+---
+# Data transformation
+
```

Comment:
Add a separate line here: "Important: This feature is in technical preview; a cluster with this feature enabled is not yet supported for production. "